pluma/pluma-view.c

```diff
@@ -1280,6 +1321,66 @@ search_window_button_press_event (GtkWidget      *widget,
 	return FALSE;
 }
 
+static void
+extension_added (PeasExtensionSet *extensions,
+		 PeasPluginInfo   *info,
+		 PeasExtension    *exten,
+		 PlumaView        *view)
+{
+	pluma_view_activatable_activate (PLUMA_VIEW_ACTIVATABLE (exten));
+}
+
+static void
+extension_removed (PeasExtensionSet *extensions,
+		   PeasPluginInfo   *info,
+		   PeasExtension    *exten,
+		   PlumaView        *view)
+{
+	pluma_view_activatable_deactivate (PLUMA_VIEW_ACTIVATABLE (exten));
+}
+
+static void
+pluma_view_realize (GtkWidget *widget)
+{
+	PlumaView *view = PLUMA_VIEW (widget);
+
+	GTK_WIDGET_CLASS (pluma_view_parent_class)->realize (widget);
+
+	g_signal_connect (view->priv->extensions,
+	                  "extension-added",
+	                  G_CALLBACK (extension_added),
+	                  view);
+	g_signal_connect (view->priv->extensions,
+	                  "extension-removed",
+	                  G_CALLBACK (extension_removed),
+	                  view);
```

Comment:
It's customary to write the name of the signal also on the first line. Test:
```
$ git grep g_signal_connect
```

Reply:
done